stock_scheduler_assignation_horizon/models/res_config_settings.py

```diff
@@ -27,3 +40,13 @@ def _check_moves_assignation_horizon(self):
         for record in self:
             if record.moves_assignation_horizon < 0:
                 raise ValidationError(_("The assignation horizon cannot be negative"))
+
+    def get_global_is_moves_assignation_limited(self):
+        get_param = self.env["ir.config_parameter"].sudo().get_param
+        param = "s_scheduler_assignation_horizon.global_moves_assignation"
+        return safe_eval(get_param(param, False))
```

Comment:
no need for safe_eval, pls use `str2bool`

Reply:
Also, these methods should not stay here. I would move them to where you need them: on the proc group.